content/blog/query-kubernetes/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Introducing CloudQuery
+authors: ["joe-duffy"]
+tags: ["Pulumi-News", "Kubernetes"]
+meta_desc: "Today we announced Pulumi CloudQuery, an SDK for programmatically querying cloud resoures."
+date: "2019-11-14"
+meta_image: "pulumi-crosswalk-k8s.png"
+---
+
+Today, as part of our Crosswalk for Kubernetes announcement[link], we're introducing CloudQuery for
+Kubernetes.
+
+CloudQuery is a relational TypeScript SDK for querying live Kubernetes resources. It makes it easy
```

Comment:
I feel like we need a little more setup about the problem before we dive into the solution.  Or perhaps after a single sentence of "CloudQuery is... ", backup to articulate the problem for a paragraph, and then dive into the solution.

Reply:
took a shot at this. but I don't really know what you had in mind.